cypress/integration/dashboard_spec.js

```diff
@@ -27,10 +17,10 @@ describe('The Dashboard Page', function () {
     /* WHEN GAME HAS STARTED */
     describe('after start of game', () => {
         it("after the start button has been pressed", () => {
-            cy.get('#start-game').click();
-            cy.get('#round-number').should('exist');
-            cy.get('#current-player').should('exist');
-            cy.get('#message').should('not.exist');
-        });
-    });
-});
+            cy.startGame();
+            cy.get('#round-number').should('exist')
+            cy.get('#current-player').should('exist')
+            cy.get('#message').should('not.exist')
+        })
+    })
+})
```

Comment:
are semi-colons being removed on purpose?

Reply:
No, missed these, will add them in now